sql/core/src/test/scala/org/apache/spark/sql/internal/SQLConfSuite.scala

```diff
@@ -17,13 +17,36 @@
 
 package org.apache.spark.sql.internal
 
+import org.scalatest.BeforeAndAfterAll
+
 import org.apache.spark.sql.{QueryTest, Row, SparkSession, SQLContext}
+import org.apache.spark.sql.execution.WholeStageCodegenExec
 import org.apache.spark.sql.test.{SharedSQLContext, TestSQLContext}
 
-class SQLConfSuite extends QueryTest with SharedSQLContext {
+class SQLConfSuite extends QueryTest with SharedSQLContext with BeforeAndAfterAll {
```

Comment:
I'm not sure if this is the proper suite to test this, how about OptimizeCodegenSuite?